crates/oxc_semantic/src/dot.rs

```diff
@@ -45,7 +45,7 @@ impl DebugDotContext<'_, '_> {
             AstKind::NumericLiteral(lit) => Some(lit.value.to_string()),
             AstKind::BooleanLiteral(lit) => Some(lit.value.to_string()),
             AstKind::StringLiteral(lit) => Some(lit.value.to_string()),
-            AstKind::BigIntLiteral(lit) => Some(lit.raw.to_string()),
+            AstKind::BigIntLiteral(lit) => Some(lit.value.to_string()),
```

Comment:
Anyone have any idea if this change is correct? For bigint `0x1_0n`, previously would have returned `"0x1_0n"`, now it returns `"16"`.

But I have no idea what this `DebugDotContext` thing is! (I assume part of CFG, but that's a part of the codebase I know nothing about)

Reply:
i think this is fine.

this function is just a debug representation of the CFG and isn't used anywhere